features/cellular/framework/AT/AT_CellularSMS.cpp

```diff
@@ -990,7 +990,7 @@ void AT_CellularSMS::add_info(sms_info_t *info, int index, int part_number)
         return;
     }
     sms_info_t *current = _sms_info;
-    sms_info_t *prev;
+    sms_info_t *prev=NULL;
```

Comment:
This may fail on astyle check. 
```C
sms_info_t *prev = NULL:
```
will pass.

Reply:
thanks - already fixed on second commit